### PR TITLE
Added cancancan gem

### PR DESCRIPTION
**Pull request by PetroKuida**
Added cancancan gem and generated ability.rb

### CODE REVIEW

**Comment by yashka713** [comment]
@PeterKujda 
Hello! Why did our build fail? Please, fix it.

File: app/models/ability.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user); end
```
Comment (DenisUA) [inline app/models/ability.rb:4]: Does it work?

**Comment by DenisUA** [comment]
Please fix the CI

**Comment by Vitaliy-Maznyi** [comment]
CI can't be fully integrated until db is created.

**Review by Vitaliy-Maznyi — APPROVED** [review]

File: app/models/ability.rb (hunk 1 above)
Comment (PetroKuida) [reply app/models/ability.rb:4]: Yes, as far as I know. Why shouldn't it?

**Pull request merged**